docker-push/action.yml

```
@@ -2,23 +2,23 @@ name: docker-push
 description: Push a Docker image to GAR.
 
 inputs:
-  gar_location:
-    description: GCP region where target GAR is located
-    required: true
-    default: us
-  should_authenticate_to_ghcr:
-    description: Whether or not to authenticate to Github Container Registry
-    required: false
-    default: "false"
   image_tags:
-    description: Newline-delimited list of images to be pushed. Typically generated by mozilla-it/deploy-actions/build-image or docker/metadata-action.
+    description: Newline-delimited list of images to be pushed.
```

We want to include what the shape of the tags needs to be. Maybe something like this:

```suggestion
    description: Newline-delimited list of images to be pushed. The docker-build action will generate this. If you don't use docker-build, you'll need to set this yourself using the following convention:<br><br>ghcr.io/mozilla/REPO/SERVICENAME:IMAGETAG<br><br>us-docker.pkg.dev/PROJECT_ID/GAR_PATH/SERVICE_NAME:IMAGE_TAG
```

---

Link to the documentation for how things are stored in GAR and GHRC would work fine here, too.

---

[c854c61](https://github.com/mozilla-it/deploy-actions/pull/54/commits/c854c61a798df7d2b269a080ac0a7f13d2e2f4e8) for posterity